mtags/src/main/scala/scala/meta/internal/mtags/ScalaToplevelMtags.scala

```diff
@@ -38,7 +38,7 @@ class ScalaToplevelMtags(
     val input: Input.VirtualFile,
     includeInnerClasses: Boolean
 ) extends MtagsIndexer {
-  private val scanner = new LegacyScanner(input, dialects.Scala212)
```

Comment:
This is not strictly related to the issue at hand, but it felt consistent to update it